backend/tests/engine/projects/conftest.py

```diff
@@ -77,3 +83,98 @@ def data_for_quadratic_funding(projects):
     matched_rewards = 35000
 
     return allocations, matched_rewards
+
+
+@pytest.fixture
+def dataset_1_for_capped_quadratic_funding(projects):
+    matched_rewards = 35000
+    allocations = [
+        AllocationItem(projects[0], 1000),
+        AllocationItem(projects[1], 2000),
+        AllocationItem(projects[2], 3000),
+        AllocationItem(projects[3], 4000),
+        AllocationItem(projects[4], 5000),
+        AllocationItem(projects[5], 6000),
+        AllocationItem(projects[6], 7000),
+        AllocationItem(projects[7], 8000),
+        AllocationItem(projects[8], 9000),
+        AllocationItem(projects[9], 10000),
+    ]
+
+    return matched_rewards, allocations
+
+
+@pytest.fixture
+def dataset_2_for_capped_quadratic_funding(projects):
+    matched_rewards = 35000
+    allocations = [
+        AllocationItem(projects[0], 1000),
+        AllocationItem(projects[1], 2000),
+        AllocationItem(projects[2], 3000),
+        AllocationItem(projects[3], 4000),
+        AllocationItem(projects[4], 5000),
+        AllocationItem(projects[5], 6000),
+    ]
+
+    return matched_rewards, allocations
+
+
+@pytest.fixture
+def many_allocations_per_project_for_capped_quadratic_funding(projects):
+    matched_rewards = 100000
+    allocations = [
+        AllocationItem(projects[0], 1000),
+        AllocationItem(projects[0], 1000),
+        AllocationItem(projects[1], 2000),
+        AllocationItem(projects[1], 2000),
+        AllocationItem(projects[2], 3000),
+        AllocationItem(projects[2], 3000),
+        AllocationItem(projects[3], 4000),
+        AllocationItem(projects[3], 4000),
+        AllocationItem(projects[4], 5000),
+        AllocationItem(projects[4], 5000),
+        AllocationItem(projects[5], 6000),
+        AllocationItem(projects[5], 6000),
+    ]
+
+    return matched_rewards, allocations
+
+
+@pytest.fixture
+def matched_rewards_with_capped_distribution(projects):
+    matched_rewards = 10000
+    computed_matched_rewards = [
```

Comment:
https://chat.wildland.dev/wildland/pl/u4tnbi6ndbdj5e9khzn5orfpko

Reply:
@leoni-q i've created a task for the corner case: https://linear.app/golemfoundation/issue/OCT-1682/handle-leftovers-from-capped-mr 

i'd leave the values you suggested as it is since that's a minor thing and we have other hurrying us up things